src/code/sched.c

```diff
@@ -420,12 +574,19 @@ void Sched_HandleRDPDone(SchedContext* sc) {
     }
 }
 
-void Sched_SendEntryMsg(SchedContext* sc) {
+/**
+ * Called by other threads in order to wake the scheduler up immediately to enqueue and
+ * possibly run a task that has been sent to the task queue. Otherwise, any pending tasks
+ * will be enqueued at VI retrace.
```

Comment:
"VI retrace" sounds weird, "vertical interrupt retrace" ?

or is it "video interface retrace"? also sounds weird but probably ok technically

unfortunate acronym all around...

Reply:
Changed "VI Retrace" to "Vertical Retrace", hopefully it's clearer now